modules/preview/preview.php

```diff
@@ -136,10 +127,12 @@ public function allow_preview_result( $posts, $query ) {
 		return $posts;
 	}
 
+	// Public API
```

Comment:
Because we're now interacting with modules directly (`Preview::get_link_expiration_options()`), it's good to separate `public` functions that are filters (not part of the public API), and `public` functions that are meant to be called from outside modules. I like to put these at the bottom of the class, because `init()` and `init()`-related filters make more sense to me at the top of the class.

Reply:
That's a good idea. I added a full PHPDoc header and `@access public` in this commit: https://github.com/Automattic/vip-workflow-plugin/pull/31/commits/81074a3fef0313f0f58befc01c366d917219ca76. We may also want to explicitly set a non-public access for other methods we use, like filter callbacks, but I haven't implemented that for this PR.